pages/index.vue

```diff
@@ -13,11 +13,12 @@
           perspiciatis ipsam ullam a repellat!
         </p>
       </div>
-      <transition name="fade">
-        <ul
-          v-show="isListVisible"
```

Comment:
Variable `isListVisible` is used so that the list is visible only when the projects are displayed and not on the hero section.
Both methods `setDotStatus()` and `displayList()` manipulate the value of this variable. I think it would be better if `v-show` checks for this property rather than `$fetchState.pending` to prevent the list being visible over the Hero section.

Reply:
I think we can do an `&`  do deal with this

Reply:
Yes, that would surely work!
However, the list functionality in the deploy preview seems to be broken. They are not acquiring the dynamic `is-active` class (by the `setDotStatus` method) and they are not navigating to a particular project when clicked.
I think the methods `scrollToSection` and `setDotStatus` need to be refactored according to the current changes.

Reply:
now i see the v-if show be used for the section below and not here also transition block wont work since its a v-show and not v-if so I added v-if below and removed the transition block here